docs/mine/lotus/split-markets-miners.md

```diff
@@ -482,3 +482,13 @@ As soon as the message is confirmed, clients will know to look for the node iden
 ```shell
 lotus-miner actor set-addrs <NEW_MULTIADDR>
 ```
+9. update the Addresses section of the markets config.toml,Update this section as same  as  mining/sealing/proving node
```

Comment:
```suggestion

9. Update the `Addresses` section of the markets `config.toml` to be the same as same the mining/sealing/proving node.
```

- Add a space after each code block.
- Capitalize the start of sentences.
- Surround variable names like `Addresses` in back ticks (`).
- Surround file names like `config.toml` in back ticks (`).
- End sentences with a period/full-stop (.).